net/sslscan/Portfile

```diff
@@ -19,9 +19,9 @@ license         {GPL-3+ OpenSSLException}
 
 depends_lib     port:openssl
```

Comment:
Maybe use the `openssl` portgroup?

Reply:
I’m not familiar with that PortGroup and I don’t see it documented on https://guide.macports.org/chunked/reference.portgroup.html. What benefit would using that PortGroup bring for this port?